Parsers/ASimFileEvent/Parsers/imFileEvent.yaml

```diff
@@ -29,4 +29,5 @@ ParserQuery: |
     vimFileEventMicrosoftSysmon,
     vimFileEventAzureTableStorage,
     vimFileEventMicrosoftWindowsEvents,
-    vimFileEventNative
\ No newline at end of file
+    vimFileEventNative,
```

Comment:
I think we updated this file as part of SentinelOne PR, please copy and update from the latest

Reply:
Done

Reply:
parameters missing

Reply:
Added now